src/get_estimation.py

```diff
@@ -695,21 +671,19 @@ def get_estimation(x, t, m, y, estimator, config):
                 calibration=True,
                 calib_method="isotonic",
             )
-        else:
-            effects = [np.nan] * 5
-    if estimator == "simulation_based":
+    elif estimator == "simulation_based":
         if config in (0, 1, 2):
             effects = r_mediate(y, t, m, x, interaction=False)
-        else:
-            effects = [np.nan] * 5
-    if estimator == "DML_huber":
+    elif estimator == "DML_huber":
         if config > 0:
             effects = medDML(y, t, m, x, trim=0.0, order=1)
-        else:
-            effects = [np.nan] * 5
-    if estimator == "G_estimator":
+    elif estimator == "G_estimator":
         if config in (0, 1, 2):
             effects = g_estimator(y, t, m, x)
-        else:
-            effects = [np.nan] * 5
+    else:
+        raise NotImplementedError("Unrecognized estimator label.")
```

Comment:
this should be a ValueError (unauthorized value for the estimator), it's not a matter that the estimator is not implemented yet.

Reply:
I'd tend to agree too

Still @bthirion proposed NotImplementedError in the previous conversation (I just unresolved it)
> NotImplementedError. You need to inform the user that he's not using the code correctly.

Reply:
done, 1493fd0